worker/auditconfigupdater/manifold_test.go

```diff
@@ -0,0 +1,242 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package auditconfigupdater_test
+
+import (
+	"time"
+
+	"github.com/juju/errors"
+	"github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	"github.com/juju/utils/set"
+	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/worker.v1"
+
+	"github.com/juju/juju/agent"
+	"github.com/juju/juju/core/auditlog"
+	"github.com/juju/juju/state"
+	statetesting "github.com/juju/juju/state/testing"
+	coretesting "github.com/juju/juju/testing"
+	"github.com/juju/juju/worker/auditconfigupdater"
+	"github.com/juju/juju/worker/dependency"
+	dt "github.com/juju/juju/worker/dependency/testing"
+	"github.com/juju/juju/worker/workertest"
+)
+
+type manifoldSuite struct {
+	statetesting.StateSuite
```

Comment:
no new worker tests should use StateSuite
or is it too hard to mock everything?

Reply:
I don't think it'll be hard to mock it out - I'll do that.

Reply:
Hmm actually, since the state tracker returns a `*state.StatePool` it's really awkward to mock out.